### DISCUSSION TITLE
Multiple color spaces not working

### DISCUSSION THREAD

**Discussion by AgentCosmic** [uga-rosa/ccc.nvim#93] [Q&A]
I copied the code from https://github.com/uga-rosa/ccc.nvim/wiki/Use-multiple-color-spaces-simultaneously
But I still see only rgb mode available.
Is there something else I need to do?

**Comment by uga-rosa** [comment]
I checked and it works fine when copied. Please make sure that you are copying everything properly and that you are not calling setup again afterwards to overwrite the settings.

**Comment by AgentCosmic** [comment]
Turns out I had to put the config under the `config` section instead of the `setup` section is lazy.nvim.
Btw, why not make this feature built-in instead of needing to copy/paste it? Also is it possible to support other color spaces like LAB?

**Comment by uga-rosa** [comment]
The reason it is not built-in is because it is designed to allow any combination of color spaces you like. One of the features of this plug-in is that users can assemble and use modules as they like, according to their own needs.
